.github/workflows/benchmarks.yml

```
@@ -0,0 +1,107 @@
+on:
+  workflow_dispatch:
+  pull_request:
```

```suggestion
  pull_request:
  push:
    branches-ignore:
      - 2.x
```

In case you have a long-running feature branch again?